.gitignore

```diff
@@ -103,6 +103,19 @@ venv.bak/
 .spyderproject
 .spyproject
 
+# VScode 
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+
+# Built Visual Studio Code Extensions
+*.vsix
```

Comment:
Is this relevant to this PR? I think no need to push these changes to .gitignore

Reply:
This is essentially the first full PR for this repo so either we get this in or splinter it as a separate PR to be expedited

Reply:
Ok if you feel this is needed it can go in this one.